lesson08/assignment/vue-cli/src/views/testmap.vue

```diff
@@ -0,0 +1,77 @@
+<template>
```

Comment:
If this view isn't being used (references to it are commented out elsewhere), no need to keep the file.